app/components/post_list/thread_overview/index.ts

```diff
@@ -24,7 +24,7 @@ const enhanced = withObservables(
                 pipe(
                     switchMap((pref) => of$(Boolean(pref[0]?.value === 'true'))),
                 ),
-            repliesCount: queryPostsInThread(database, rootId).observeCount(),
+            repliesCount: queryPostRepliesCount(database, rootId).observeCount(false),
```

Comment:
Disabling throttle because it gets updated immediately from the API call before the throttle and the query is skipping the update.

Reply:
I still think something shady is going on here and we should look into it, but it is OK for now.